ilc/server/tailor/parse-override-config.js

```diff
@@ -0,0 +1,36 @@
+const _ = require('lodash');
+const CIDRMatcher = require('cidr-matcher');
+
+var privateNetworks = new CIDRMatcher([
+    '10.0.0.0/8',
+    '192.168.0.0/16',
+    '172.16.0.0/12',
+    '127.0.0.0/8',
+]);
+
+const isPrivateNetwork = link => {
+    const matchedIp = link.match(/\b\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}\b/);
+    return matchedIp && matchedIp[0] && privateNetworks.contains(matchedIp[0]);
+}
+const isLink = text => typeof text === 'string' && /https?:\/\/?/.test(text);
```

Comment:
Is it possible that we may have a link without a protocol?

Reply:
perhaps it can be, but I decided that it's better to force developers to set whole url.

Reply:
> I guess we should use `^` here at the beginning `/^https` because if somebody is going to provide `asdsadsahttps://asdsad/` then it is going to be true

we shouldn't do it, because it's vulnerability, simple example ` 
  https://spoofedlink.com/`

Reply:
> I guess we should use `^` here at the beginning `/^https` because if somebody is going to provide `asdsadsahttps://asdsad/` then it is going to be true

we shouldn't do it, because it's vulnerability, simple example with space, which is correct url " 
  https://spoofedlink.com/"